x-pack/libbeat/tests/system/test_management.py

```diff
@@ -13,7 +13,9 @@
 from base import BaseTest
 
 
-INTEGRATION_TESTS = os.environ.get('INTEGRATION_TESTS', False)
+# Disable because waiting artifacts from https://github.com/elastic/kibana/pull/31660
+# INTEGRATION_TESTS = os.environ.get('INTEGRATION_TESTS', False)
+INTEGRATION_TESTS = False
```

Comment:
chicken-egg, we need snapshots of kibaba and ES to have the integration suite working. So I am disabling it.